components/builder-originsrv/src/migrations/2018-06-29-222627_auto_build_opt_in/up.sql

```diff
@@ -0,0 +1,69 @@
+ALTER TABLE IF EXISTS origin_projects ADD COLUMN IF NOT EXISTS auto_build bool NOT NULL DEFAULT true;
```

Comment:
I thought this was supposed to be off by default?

Reply:
Yes that was the original plan.. however in the interest of not breaking any existing build workflows (eg, all core plans auto builds would immediately stop), it seems better to default it to true in the DB, and then have a migration path for turning off unneeded builds.  Note that the UI will default auto build to Off as default, so for new plan connections we will have the more conservative behavior.